<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtoPanel</class>
 <widget class="QWidget" name="ProtoPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Protocol generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../resources/nicos-gui.qrc">
    <normaloff>:/paste</normaloff>:/paste</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="5">
       <widget class="QPushButton" name="genBtn">
        <property name="text">
         <string>Generate</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/nicos-gui.qrc">
          <normaloff>:/down</normaloff>:/down</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Last run:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>First run:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ShadowTextLineEdit" name="lastEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="ShadowTextLineEdit" name="firstEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="stampBox">
        <property name="text">
         <string>With timestamps</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="printBtn">
          <property name="text">
           <string>Print</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../resources/nicos-gui.qrc">
            <normaloff>:/print</normaloff>:/print</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Font size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fontBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="text">
           <string>Save</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../resources/nicos-gui.qrc">
            <normaloff>:/save</normaloff>:/save</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="outText">
        <property name="font">
         <font>
          <family>Monospace</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/previous</normaloff>:/previous</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/next</normaloff>:/next</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/refresh</normaloff>:/refresh</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionAddComment">
   <property name="text">
    <string>Add comment</string>
   </property>
   <property name="toolTip">
    <string>Add a free-form comment entry to the logbook</string>
   </property>
  </action>
  <action name="actionAddRemark">
   <property name="text">
    <string>Set Remark</string>
   </property>
   <property name="toolTip">
    <string>Set the remark (instrument configuration header) and add it to the logbook</string>
   </property>
  </action>
  <action name="actionAttachFile">
   <property name="text">
    <string>Attach file</string>
   </property>
   <property name="toolTip">
    <string>Attach a file to the logbook</string>
   </property>
  </action>
  <action name="actionNewSample">
   <property name="text">
    <string>New Sample</string>
   </property>
   <property name="toolTip">
    <string>Enter a new sample name</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/print</normaloff>:/print</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShadowTextLineEdit</class>
   <extends>QLineEdit</extends>
   <header>nicos.clients.gui.widgets.lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections/>
</ui>
